app/src/main/java/fr/free/nrw/commons/nearby/NearbyMapFragment.java

```diff
@@ -489,6 +493,21 @@ public void onSlide(@NonNull View bottomSheet, float slideOffset) {
             commonsButtonText.setVisibility(View.GONE);
             directionsButtonText.setVisibility(View.GONE);
         }
+        title.setOnLongClickListener(view -> {
+                    ClipboardManager clipboard = (ClipboardManager) getContext().getSystemService(Context.CLIPBOARD_SERVICE);
```

Comment:
Move to `Utils.copy(text)`. 

Also, `MediaDetailFragment:onCopyWikicodeClicked` also uses clipboard. You can replace the code there with your util method.

Reply:
I'll make the changes